openqa-trigger/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.opensuse.org/home/vpereirabr/dockerimages/containers/obs-tools/base:latest
+
+RUN zypper --non-interactive --quiet ref \
+ && zypper -n in curl openQA-client
+
+COPY . /home/frontend/openqa-trigger
```

Comment:
Why COPY, why not mount the checkout?

Reply:
Hm you even mount in the Makefile below....

Reply:
Done with the `docker-compose.yml` file.